README.md

```diff
@@ -63,38 +63,48 @@ When you run the pipeline, you will use the mandatory `--input` parameter to spe
 
-***Note***: You must specify the full path for the reference fasta, even if it is in your working directory.
+Documented below are the two different methods by which you can generate the interval files for Mutect2 when you run the pipeline using the nextflow command.  
+
+***Generate optimised number of intervals based on your genome size***  
+When you run the pipeline, you will use the mandatory --ref  and --dict parameters to specify the location and name of the reference files. The genomic-interval files will be generated by the pipeline based on these two parameters.
+```
+--ref /path/to/ref.fasta --dict /path/to/ref.dict
+```
 
+***Generate a specific number of intervals (Optional)***   
+You can define a specific number genomic-intervals based on the number of cpus available to parallelize execution. When you run the pipeline, you will use the optional `--number_of_intervals` parameter as an integer value. The genomic-interval files will be generated by the pipeline based on this parameter.
 
-### 3. Download files
-#### Download the sub-interval files required for Mutect2 
-  - **Step 1**: Click on the [link](https://console.cloud.google.com/storage/browser/genomics-public-data/resources/broad/hg38/v0;tab=objects?pli=1&prefix=&forceOnObjectsSortingFiltering=false&pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22)))
-  - **Step 2**: Click on [scattered_calling_intervals] and select all checkboxes and click `Download`. 
-  - **Step 3**: To download all the files, you will need to install the utility `gsutil`. Please follow the steps as shown below in a command-line window
-    - Download, unzip and install the excecutable  
-      - curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-cli-457.0.0-linux-x86_64.tar.gz
-      - tar -xf google-cloud-cli-457.0.0-linux-x86_64.tar.gz
-      - ./google-cloud-sdk/install.sh
-      - Close rhe current command-line window and re-open it. The utility `gsutil` will now be available in the path.
-  - **Step 4**:Go back to Step (2) and download the files as suggested.
 
-#### A Panel of Normals (PoN) for Mutect2  
-The user can create a PoN file using our [pipeline](https://github.com/Sydney-Informatics-Hub/Somatic-ShortV?tab=readme-ov-file#4-create-pon-per-genomic-interval).
+#### Create a Panel of Normals (PoN) for Mutect2  
+The user can create a a panel of normals (PoN) containing germline and artifactual sites for use with Mutect2 using the instructions provided [here](https://gatk.broadinstitute.org/hc/en-us/articles/360037058172-CreateSomaticPanelOfNormals-BETA-). We will include this functionality as an optional module in the next version of the pipeline.
```

Comment:
```suggestion
You will need to provide a [panel of normals (PoN)](https://gatk.broadinstitute.org/hc/en-us/articles/360037058172-CreateSomaticPanelOfNormals-BETA-) containing germline and artifactual sites for use with Mutect2. This functionality is currently under development and will be included in this pipeline shortly.  
```

Reply:
Ensure the version of GATK documentation you're linking to corresponds with the version of GATK being run by your pipeline

Reply:
Changed the link to gatk4.4